ggshield/core/config/user_config.py

```diff
@@ -203,10 +204,15 @@ def is_ghsa_valid(ghsa_id: str) -> bool:
     return bool(GHSA_ID_PATTERN.fullmatch(ghsa_id))
 
 
+def is_cve_id_valid(cve_id: str) -> bool:
+    return bool(CVE_ID_PATTERN.fullmatch(cve_id))
+
+
 def validate_vuln_identifier(value: str):
-    if not is_ghsa_valid(value):
+    if not is_ghsa_valid(value) and not is_cve_id_valid(value):
```

Comment:
nit

```suggestion
    if not (is_ghsa_valid(value) or is_cve_id_valid(value)):
```

I'm not a big fan of `not ... and not ...` but doesn't matter